src/Globals.h

```diff
@@ -16,13 +16,25 @@ enum class RoverState {
 	LookingForTennisball,
 };
 
+enum class DriveMode {
+	Normal,
+	TurnInPlace,
+	Crab,
+};
+
+static const std::map<DriveMode, std::string> driveModeStrings = {
+	{DriveMode::Normal, "Normal"},
+	{DriveMode::TurnInPlace, "TurnInPlace"},
+	{DriveMode::Crab, "Crab"}};
+
 // NOTE(sasha): To keep linker happy, declare globals with "extern" here and then
 //              provide variable definition in Globals.cpp
 namespace Globals {
 /**
    @deprecated
  */
 extern RoverState curr_state;
+extern std::tuple<DriveMode, bool> driveMode;
```

Comment:
Can we add documentation on what the parts of the drive mode are? Particularly the `bool`, which is vague